mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule RailwayIpc.MixProject do
   def project do
     [
       app: :railway_ipc,
-      version: "0.2.1",
+      version: "0.2.2",
```

Comment:
I'm being a little lazy here and not calling this 0.3.0 even though it technically is adding a new feature ([symantic versioning](https://semver.org/#summary)). It _is_ backwards compatible though!

If we _do_ want to go 0.3.0 then I'll just need to update railway UI as well.